src/models/annotation.model.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const DynamicsDalService = require('../services/dynamicsDal.service')
+const BaseModel = require('./base.model')
+const LoggingService = require('../services/logging.service')
+const Utilities = require('../utilities/utilities')
+
+module.exports = class Annotation extends BaseModel {
+  constructor (annotation) {
+    super()
+    this.entity = 'annotations'
+    this.id = annotation.id
```

Comment:
Needs an '''if (annotation) around this as per the other model classes, in case the passed in annotation is undefined.